registry/handlers/manifests.go

```diff
@@ -322,7 +335,13 @@ func (imh *manifestHandler) PutManifest(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	isAnOCIManifest := mediaType == v1.MediaTypeImageManifest || mediaType == v1.MediaTypeImageIndex
+	var isAnOCIManifest bool
+	switch mediaType {
+	case v1.MediaTypeImageManifest, v1.MediaTypeArtifactManifest, v1.MediaTypeImageIndex:
+		isAnOCIManifest = true
+	default:
+		isAnOCIManifest = false
+	}
```

Comment:
nit: simplify code 😄  
```suggestion
	isAnOCIManifest := false
	switch mediaType {
	case v1.MediaTypeImageManifest, v1.MediaTypeArtifactManifest, v1.MediaTypeImageIndex:
		isAnOCIManifest = true
	}
```

Reply:
changed accordingly.